packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -822,6 +854,76 @@ export function getCustomInputsResponses(
   return customInputsResponses;
 }
 
+/** Updates the evt object with video call data found from booking references
+ *
+ * @param bookingReferences
+ * @param evt
+ *
+ * @returns updated evt with video call data
+ */
+export const addVideoCallDataToEvt = (bookingReferences: BookingReference[], evt: CalendarEvent) => {
```

Comment:
Considering other functions already have the `event` name, could we rename this to `addVideoCallDataToEvent`? Two extra characters won't hurt.

Reply:
Addressed in https://github.com/calcom/cal.com/pull/12905/commits/649dd1fd0d6df98dfe77424b7713b7bf2aa178e0